src/main/resources/i18n/strings.properties

```diff
@@ -142,6 +142,9 @@ unlock.error.customPath.description.hideawayNotDir=The temporary, hidden file "%
 unlock.error.customPath.description.couldNotBeCleaned=Your vault could not be mounted to the path "%s". Please try again or choose a different path.
 unlock.error.customPath.description.notEmptyDir=The custom mount path "%s" is not an empty folder. Please choose an empty folder and try again.
 unlock.error.customPath.description.generic=You have selected a custom mount path for this vault, but using it failed with the message: %2$s
+unlock.error.fuseRestartRequired.title=Unlock "%s" failed
```

Comment:
i would use a more generic key for the window title, because unlock failed can have many causes. How about `unlock.error.title`?